data_science/datasciencetopics.md

```diff
@@ -0,0 +1,65 @@
+# Data Science Topics and Subtopics
+
+### Terminal
+- Directories
+- Terminal commands
+- Installation and setup
+- Jupyter notebook
+
+### Git and GitHub
+- Cloning Repositories
+- Version Control
+- Branching, pull requests, working with a team
+
+### Markdown
+- Intro to basic syntax (i.e. headings, lists, code blocks, embedding links and images)
+
+### Progrmming Concepts
+- Iteration
+- Conditional Statements
+- Sequences (Array, List)
+- Data Types (Strings, numbers, logical)
+
+### Python 1 - Intro to Python
+- Integrating programming concepts
+- Dictionaries
+- Lists

```

Comment:
Consider broadening the topic to contain other builtin containers in python (tuples, sets)?